sdks/java/io/hadoop/jdk1.8-tests/pom.xml

```diff
@@ -30,8 +30,52 @@ and overrides the enforcer rules to allow 1.8 only behavior without making all o
   </parent>
   <artifactId>jdk1.8-tests</artifactId>
   <name>Apache Beam :: SDKs :: Java :: IO :: Hadoop :: jdk1.8-tests</name>
+  
   <build>
+    <pluginManagement>
+      <plugins>
+        <!-- BEAM-1661 -->
```

Comment:
this JIRA refers to jdbc - shouldn't be included here.

Please add a comment explaining why you're handling this differently from the rest of the project's guava shading.